src/main/sampling/Sample.cpp

```diff
@@ -182,6 +182,122 @@ namespace lsp
             return true;
         }
 
+        bool Sample::stretch(size_t channels, int stretch_samples, size_t chunk_size)
```

Comment:
```
ssize_t stretch_samples
```

Reply:
I don't think `ssize_t` is a suitable type since `stretch_samples` can be a negative number smaller than -1. It will be subtracted from current sample length to get the desired length. Which means that the desired length will increase if this number is negative.

Reply:
`ssize_t` is a `signed size_t`, so this is a good replacment for `int`.